bundle/manifests/gitops-operator.clusterserviceversion.yaml

```diff
@@ -596,30 +596,13 @@ spec:
                 control-plane: argocd-operator
             spec:
               containers:
-              - args:
-                - --secure-listen-address=0.0.0.0:8443
-                - --upstream=http://127.0.0.1:8080/
-                - --logtostderr=true
-                - --v=10
-                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
-                name: kube-rbac-proxy
-                ports:
-                - containerPort: 8443
-                  name: https
-                resources: {}
-              - args:
-                - --health-probe-bind-address=:8081
-                - --metrics-bind-address=127.0.0.1:8080
-                - --leader-elect
-                command:
-                - /manager
```

Comment:
@iam-veeramalla is this removal intentional?

Reply:
Yes Jaideep, I am trying to keep the CSV similar to the downstream one and I realised in the downstream we don't use this. 

If you build the bundle for this operator locally, it create the operator pod with 2 containers .. a proxy container which we don't use and the manager container. 

We don't use the proxy container in the downstream.. so just to keep them aligned I removed this.